### PR TITLE
Removal of Jenkins

### PR DESCRIPTION
**Pull request by sadovnikov**
new CI setup:

minimesos ci dataflow

### CODE REVIEW

File: minimesos/src/main/java/com/containersol/minimesos/docker/DockerClientFactory.java [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
     private static DockerClient dockerClient;
 
+    private final static String bad_example = "bad example";
```
Comment (containersol) [inline minimesos/src/main/java/com/containersol/minimesos/docker/DockerClientFactory.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)
MAJOR Remove this unused "bad_example" private field. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1068)
MINOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS00115)

**Comment by containersol** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 1 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by sadovnikov** [comment]
Configuration through Travis CI environment variables:

image

The added code (with different branch name) was tested:

image

and it resulted in adding the scan to https://ci.infra.container-solutions.com/sonar/project/history?id=minimesos-project

image

**Comment by alex-glv** [comment]
:+1:

**Comment by remmelt** [comment]
👍

**Pull request merged**